.gitignore

```diff
@@ -25,3 +25,9 @@ replay_pid*
 .idea/
 target/
 *.iml
+.classpath
+.factorypath
+.project
+.settings/org.eclipse.core.resources.prefs
+.settings/org.eclipse.jdt.apt.core.prefs
+.settings/org.eclipse.jdt.core.prefs
```

Comment:
Let's add the entire `bin/*` directory into this .gitignore so that we can remove these files out of the repo